spec/features/create_new_space_spec.rb

```diff
@@ -13,4 +15,21 @@
     end
   end
 
+  scenario "user can create multiple spaces" do
+    visit '/users/new'
+      sign_up(new_user)
+      click_button "Create Account"
+    visit '/spaces/new'
+    create_space
+    click_button("create space")
+    visit '/spaces/new'
+    create_second_space
```

Comment:
you could refactor the create_space and create_second_space into one method which takes args